common/types/message/message.go

```diff
@@ -52,22 +52,14 @@ type ChunkTaskDetail struct {
 
 // BatchTaskDetail is a type containing BatchTask detail.
 type BatchTaskDetail struct {
-	ChunkInfos      []*ChunkInfo     `json:"chunk_infos"`
-	ChunkProofs     []*ChunkProof    `json:"chunk_proofs"`
-	ParentStateRoot common.Hash      `json:"parent_state_root"`
-	ParentBatchHash common.Hash      `json:"parent_batch_hash"`
-	BatchHeader     *codecv3.DABatch `json:"batch_header"`
+	ChunkInfos  []*ChunkInfo     `json:"chunk_infos"`
+	ChunkProofs []*ChunkProof    `json:"chunk_proofs"`
+	BatchHeader *codecv3.DABatch `json:"batch_header"`
```

Comment:
why delete the `ParentStateRoot` and `ParentBatchHash` , no required?

Reply:
Yes, we do not require it anymore.
- The `parent_batch_hash` can be found from the `BatchHeader`
- The `parent_state_root` can be found from the first chunk's data from `ChunkInfos[0]`

Reply:
ok

Reply:
cc @colinlyguo